README.md

```diff
@@ -1,46 +1,31 @@
-# Getting Started with Create React App
+# shopping-list
+
+> A shopping list using React and Firebase 
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Running
+
+Remember to have typescript installed. You can ensure that by running:
```

Comment:
Not true, only `yarn install` is required. Instructions for installing yarn should be included